src/kili/adapters/kili_api_gateway/issue/__init__.py

```diff
@@ -18,11 +18,11 @@ class IssueOperationMixin(BaseOperationMixin):
     """GraphQL Mixin extending GraphQL Gateway class with Issue related operations."""
 
     def create_issues(
-        self, type_: IssueType, issues: List[IssueToCreateKiliAPIGatewayInput]
+        self, type_: IssueType, issues: List[IssueToCreateKiliAPIGatewayInput], description: str
     ) -> List[IssueId]:
         """Send a GraphQL request calling createIssues resolver."""
         created_issue_entities: List[IssueId] = []
-        with tqdm.tqdm(total=len(issues), desc="Creating issues") as pbar:
+        with tqdm.tqdm(total=len(issues), desc=description) as pbar:
```

Comment:
I think it would be nice here to be able to disable the pbar if needed

so we should implement what we agreed on few days ago: a ProgressBarOptions dataclass with a disable_tqdm and a description

Reply:
added in a separate ticket